index.html

```diff
@@ -47,41 +47,54 @@
         </div>
       </div>
     </nav>
-    <div class="slider-container" id="welcome">
-      <div class="ism-slider" data-transition_type="fade" id="my-slider">
-        <ol>
-          <li>
-            <img src="images/image_slider_crowd.jpg">
-            <div class="ism-caption ism-caption-0">FOSSASIA</div>
-            <div class="ism-caption ism-caption-1" data-delay="200">
-                We build a new era of Technology.<br>
-                We connect developers together.<br>
-                We are an open-source community.
-            </div>
-          </li>
-          <li>
-            <img src="images/image_slider_fossasia.jpg">
-            <div class="ism-caption ism-caption-0">Google Code-in 2016</div>
-            <div class="ism-caption ism-caption-1" data-delay="200">
-                Educating and inspiring<br>
-                students by being involved in<br>
-                open-source projects.<br>
-                <a href="//codein.withgoogle.com/organizations/fossasia/">
-                    Participate now!</a>
-            </div>
-          </li>
-          <li>
-            <img src="images/image_slider_machine.jpg">
```

Comment:
Please comment in the commit message, why you have deleted the following sliders.